src/include/event_config.h

```diff
@@ -63,6 +63,17 @@ typedef struct
     bool is_advanced;
 } event_option_t;
 
+/*
+ * struct holds
+ *   opt_name = name of the option with invalid value
+ *   error = string of the error message
+ */
+typedef struct
+{
+    char *opt_name;
+    char *error;
```

Comment:
I need to correct myself and I must apologize for not reviewing my own comments (I wrote my comment after midnight, that may explain something).

When I joined ABRT team, I was taught that the prefix of structured members is good for greping. If you use "opt_", it will not be possible to distinguish between uses of `invalid_option_t` and `event_option_t`. The same apply for the member error - there are plenty of  `error` occurrences in libreport source codes.

I suggest using the prefix `invopt_` for members of the structure `invalid_option_t`.